src/components/common/ModalDisclaimer.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <astar-default-modal
+    v-if="show"
+    :show="show"
+    :title="$t('disclaimer.disclaimer')"
+    :is-closing="isClosingModal"
+    :width="650"
+    :class="'highest-z-index'"
+    @close="decline()"
+  >
+    <div class="wrapper--modal-disclaimer">
+      <div class="row--description">
+        <div class="text--md">
+          {{ $t('disclaimer.introduce') }}
+        </div>
+        <div class="text--title">{{ $t('disclaimer.terms') }}</div>
+        <div class="text--lg">{{ $t('disclaimer.chapter1') }}</div>
+        <div class="text--md">
+          {{ $t('disclaimer.para1') }}
+        </div>
+        <div class="text--lg">{{ $t('disclaimer.chapter2') }}</div>
+        <div class="text--md">
+          {{ $t('disclaimer.para1') }}
+        </div>
+        <div class="text--lg">{{ $t('disclaimer.chapter3') }}</div>
+        <div class="text--md">
+          {{ $t('disclaimer.para2') }}
+        </div>
+        <div class="text--md">
+          {{ $t('disclaimer.closeWord') }}
+        </div>
+      </div>
+
+      <div class="container--buttons">
```

Comment:
I see dark border around buttons in light theme
<img width="606" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/8452361/9a5619ba-f33a-4e3d-9cff-7ac74ba86f2b">